sinonjs/referee-custom-assertions
# Update dependencies

This PR upgrades all the test runners to their latest versions, so the examples can be credible.

#### How to verify

1. Check out this branch
1. `npm ci`
1. `npm test`
1. Observe that all tests pass :)
